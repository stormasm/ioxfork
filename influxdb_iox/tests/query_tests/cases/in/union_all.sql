-- IOX_SETUP: TwoMeasurementsManyFields

-- Validate name resolution works for UNION ALL queries
-- IOX_COMPARE: sorted
SELECT state AS name FROM h2o UNION ALL SELECT city AS name FROM h2o;
