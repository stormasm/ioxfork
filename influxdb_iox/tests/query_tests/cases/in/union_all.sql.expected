-- Test Setup: TwoMeasurementsManyFields
-- SQL: SELECT state AS name FROM h2o UNION ALL SELECT city AS name FROM h2o;
-- Results After Sorting
+--------+
| name   |
+--------+
| Boston |
| Boston |
| Boston |
| Boston |
| CA     |
| MA     |
| MA     |
| MA     |
+--------+